code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -0,0 +1,890 @@
+	H.bleed(120)
+	var/obj/item/organ/external/chest = H.get_organ(BODY_ZONE_CHEST)
+	chest.fracture()
+	var/datum/organ/heart/datum_heart = H.get_int_organ_datum(ORGAN_DATUM_HEART)
+	var/obj/item/organ/internal/our_heart = datum_heart.linked_organ
+	our_heart.receive_damage(20, TRUE)
+
+/datum/tarot/reversed/the_chariot
+	name = "VII - The Chariot?"
+	desc = "May nothing walk past you."
+	card_icon = "the_chariot?"
+
+/datum/tarot/reversed/the_chariot/activate(mob/living/target)
+	target.Stun(4 SECONDS)
+	new /obj/structure/closet/statue/indestructible(target.loc, target)
+
+/datum/tarot/reversed/justice
+	name = "VIII - Justice?"
+	desc = "May your sins come back to torment you."
+	card_icon = "justice?"
+
+/datum/tarot/reversed/justice/activate(mob/living/target)
+	var/list/ignored_supply_pack_types = list(
+		/datum/supply_packs/abstract,
+		/datum/supply_packs/abstract/shuttle
+	)
+	var/chosen = pick(SSeconomy.supply_packs - ignored_supply_pack_types)
+	var/datum/supply_packs/the_pack = new chosen()
+	var/spawn_location = get_turf(target)
+	var/obj/structure/closet/crate/crate = the_pack.create_package(spawn_location)
+	crate.name = "magic [crate.name]"
+	qdel(the_pack)
+
+/datum/tarot/reversed/the_hermit
+	name = "IX - The Hermit?"
+	desc = "May you see the value of all things in life."
+	card_icon = "the_hermit?"
+
+/datum/tarot/reversed/the_hermit/activate(mob/living/target) //Someone can improve this in the future (hopefully comment will not be here in 10 years.)
+	for(var/obj/item/I in view(7, target))
+		if(istype(I, /obj/item/gun))
+			new /obj/item/stack/spacecash/c200(get_turf(I))
+			qdel(I)
+			continue
+		if(istype(I, /obj/item/grenade))
+			new /obj/item/stack/spacecash/c50(get_turf(I))
+			qdel(I)
+		if(istype(I, /obj/item/clothing/suit/armor))
+			new /obj/item/stack/spacecash/c100(get_turf(I))
+			qdel(I)
+		if(istype(I, /obj/item/melee/baton))
+			new /obj/item/stack/spacecash/c100(get_turf(I))
+			qdel(I)
+
+/datum/tarot/reversed/wheel_of_fortune
+	name = "X - Wheel of Fortune?"
+	desc = "Throw the dice of fate."
+	card_icon = "wheel_of_fortune?"
+
+/datum/tarot/reversed/wheel_of_fortune/activate(mob/living/target)
+	var/obj/item/dice/d20/fate/one_use/gonna_roll_a_one = new/obj/item/dice/d20/fate/one_use(get_turf(target))
+	gonna_roll_a_one.diceroll(target)
+
+/datum/tarot/reversed/strength
+	name = "XI - Strength?"
+	desc = "May you break their resolve."
+	card_icon = "strength?"
+
+/datum/tarot/reversed/strength/activate(mob/living/target)
+	for(var/mob/living/M in oview(9, target))
+		M.Hallucinate(2 MINUTES)
+		new /obj/effect/hallucination/delusion(get_turf(M), M)
+		M.adjustBrainLoss(30)
+
+/datum/tarot/reversed/the_hanged_man
+	name = "XII - The Hanged Man?"
+	desc = "May your greed know no bounds."
+	card_icon = "the_hanged_man?"
+
+/datum/tarot/reversed/the_hanged_man/activate(mob/living/target)
+	var/obj/structure/cursed_slot_machine/pull_the_lever_kronk = new /obj/structure/cursed_slot_machine(get_turf(target))
```

Comment:
There for other people to pull. Otherwise, person is stuck with the curse till they die, where as this lets people gamble on it, for the loot / to save others from the curse.

Reply:
Sounds good